client/my-sites/checkout/checkout-thank-you/domain-transfer-to-any-user/index.tsx

```diff
@@ -1,13 +1,13 @@
-import { ConfettiAnimation } from '@automattic/components';
```

Comment:
> The confetti animation removed. [UPDATE: We are keeping the animation after all p9Jlb4-7Hx-p2#comment-10513]

Looks like you updated testing instructions but forgot to update the code? Or did I miss something?

Reply:
This is added to the `<ThankYouV2>` component https://github.com/Automattic/wp-calypso/commit/538dd4e9a3a6af82c43979dac4484e0539b3be72, so we don't need to add it per page.

Reply:
Hm...
1) I don't see confetti in the browser
2) I don't see it locally after checkout to the branch 
3) Also [here](https://github.com/Automattic/wp-calypso/commit/538dd4e9a3a6af82c43979dac4484e0539b3be72) I see warning `This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.`

Are you sure that such changes committed to the `trunk` branch?

Reply:
Something weird happened while syncing with master with rebase/merge, and so the latest code wasn't pulled locally. Confirmed that the animation is working as expected!